src/modules/person/domain/person.ts

```diff
@@ -27,6 +27,7 @@ export type PersonCreationParams = {
     username?: string;
     password?: string;
     personalnummer?: string;
+    email?: string;
```

Comment:
Warum brauche ich die Email in diesem Aggregate? Ich sehe nicht, wo das verwendet wird